### PR TITLE
New create coder settings layout kovalchuk

### PR DESCRIPTION
**Pull request by Maksson**

### CODE REVIEW

**Review by MetalMari — APPROVED** [review]

**Review by OleksandrHavron — APPROVED** [review]

File: front-end/src/Components/pages/coderProfile/ConfirmPassword.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+};
+
+const RegistrationForm = () => {
+
+	const [User, setUser] = useState(initialState);
+	const [passwordError, setPasswordError] = useState(passVerificationError);
+	useEffect(() => { }, [User]);
+
+	const handleOnChange = (e) => {
+		const { name, value } = e.target;
```
Comment (vvelizariy) [inline front-end/src/Components/pages/coderProfile/ConfirmPassword.js]: Please align intendation to be consistent with other files

File: front-end/src/Components/pages/coderProfile/ConfirmPassword.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	hasLower: false,
+	hasNumber: false,
+	confirmPass: false,
+};
+
+const RegistrationForm = () => {
+
+	const [User, setUser] = useState(initialState);
+	const [passwordError, setPasswordError] = useState(passVerificationError);
+	useEffect(() => { }, [User]);
```
Comment (vvelizariy) [inline front-end/src/Components/pages/coderProfile/ConfirmPassword.js]: Is empty useEffect required?

File: front-end/src/Components/pages/coderProfile/ConfirmPassword.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	isLenthy: false,
+	hasUpper: false,
+	hasLower: false,
+	hasNumber: false,
+	confirmPass: false,
+};
+
+const RegistrationForm = () => {
+
+	const [User, setUser] = useState(initialState);
```
Comment (vvelizariy) [inline front-end/src/Components/pages/coderProfile/ConfirmPassword.js]: Please fix
const [user, setUser] = useState(initialState);

User should be used for naming object or class, simple variables should be named like `user`.
This also corresponds to exaples of usage useEffect in documentation

File: front-end/src/Components/pages/coderProfile/ConfirmPassword.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+						<Form.Text>
+							{!passwordError.confirmPass && (<div className="text-danger mb-3">Password doesn't match!</div>)}
+						</Form.Text>
+						<ul className="mt-2">
+							<li className={passwordError.isLenthy ? "text-success" : "text-danger"}> Min 8 characters</li>
+							<li className={passwordError.hasUpper ? "text-success" : "text-danger"}>At least one upper case</li>
+							<li className={passwordError.hasLower ? "text-success" : "text-danger"}> At least one lower case</li>
+							<li className={passwordError.hasNumber ? "text-success" : "text-danger"}> At least one number</li>
+						</ul>
+						<Button variant="primary" type="submit" disabled={Object.values(passwordError).includes(false)}>Submit</Button>
```
Comment (vvelizariy) [inline front-end/src/Components/pages/coderProfile/ConfirmPassword.js]: Just
disabled={passwordError.confirmPass !== false)

File: front-end/src/Components/pages/coderProfile/SettingBody.js [hunk 5]
```diff
@@ -0,0 +1,162 @@
+import { Component } from "react";
+import { Card, Button, Form, InputGroup, Col, Row } from 'react-bootstrap';
+import RegistrationForm from "./ConfirmPassword";
+import LinkedinLogo from "./images/LinkedinLogo.png"
+import TwitterLogo from "./images/TwitterLogo.png"
+import FacebookLogo from "./images/FacebookLogo.png"
```
Comment (vvelizariy) [inline front-end/src/Components/pages/coderProfile/SettingBody.js]: Keep alphabetically sorted

File: front-end/src/Components/pages/coderProfile/ConfirmPassword.js (hunk 1 above)
Comment (vvelizariy) [reply front-end/src/Components/pages/coderProfile/ConfirmPassword.js]: Actually you've left indentation as is.
Please compare tabs in this file and DataCard.js.

File: front-end/src/Components/pages/coderProfile/ConfirmPassword.js (hunk 2 above)
Comment (Maksson) [reply front-end/src/Components/pages/coderProfile/ConfirmPassword.js]: Resolved

File: front-end/src/Components/pages/coderProfile/ConfirmPassword.js (hunk 1 above)
Comment (Maksson) [reply front-end/src/Components/pages/coderProfile/ConfirmPassword.js]: Changed

**Review by vvelizariy — APPROVED** [review]

**Pull request merged**